app_server/app/adapters/socket/Room.js

```diff
@@ -75,4 +75,21 @@ Class.removeAllClients = function() {
   this.__clients = {};
 };
 
+Class.getNumberOfUsers = function() {
+  let users = {};
+  let count = 0;
+  for (var socketId in this.getClients()) {
```

Comment:
Ah ok, it works. Could be I tried it awhile back, and Mozilla's JS docs seems to still keep the `var` style for `for` loops.